README.md

```diff
@@ -134,8 +134,8 @@ tempname | tmpname
 tempdir | tmpdir 
 mktemp | mktmp 
 mktempdir | mktmpdir 
-chmod (non-recursive) | chmod (recursive unix-only)
-chown (PR) | chown (unix only)
+chmod | chmod (recursive unix-only)
```

Comment:
I guess we could remove the `recursive`  comment since the keyword has been around for a couple releases now.

Reply:
Also, do you know if the base `chmod` works on windows? I'm guessing it's using libuv?

Reply:
Yes, it calls the function of the same name provided by the OS. I suppose it's not clear though if that makes any sense, since the operation it implements is quite different: https://github.com/libuv/libuv/blob/f60713f7cf3003947159a7c44e0e7bb2eb98942b/src/win/fs.c#L1130-L1149